CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,39 @@
 # Code of Conduct
 
-Facebook has adopted a Code of Conduct that we expect project participants to adhere to. Please [read the full text](https://code.fb.com/codeofconduct/) so that you can understand what actions will and will not be tolerated.
```

Comment:
I'm not sure it's a good idea to remove this link here.

Reply:
We don't want to remove this link and dump the text here because it can get outdated. We rather have a centralized place for the most updated Code of Conduct. Please revert all changes to this file.

Reply:
Sure Thing, I get what you are saying. I"ll do it.